src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -537,3 +537,10 @@
     (:less-than-three-minutes constants/wallet-transaction-estimation) "1-3"
     (:less-than-five-minutes constants/wallet-transaction-estimation)  "3-5"
     ">5"))
+
+(defn get-account-by-session
+  [db session]
+  (let [accounts                (get-in db [:wallet :accounts])
+        session-account-address (first (:accounts session))
+        [_ address]             (network-utils/split-network-full-address session-account-address)]
+    (get-account-by-address (vals accounts) address)))
```

Comment:
this util seems specific to wallet-connect, since you're passing the session. You could either move it to the wallet-connect ns or pass the address instead of the session (but then you already have `get-account-by-address` that does almost the same thing)